### PR TITLE
Added instructions for World Map, enlarged icons for Consumption Graph

### PR DESCRIPTION
**Pull request by VirginiaBalseiro-Zuehlke**

### CODE REVIEW

File: src/components/ConsumptionGraph.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ class ConsumptionGraph extends Component {
                             zoomin: false,
                             zoomout: false,
                             pan: false,
-                            reset: true | '<img src="/static/icons/reset.png" width="20">',
+                            reset: true  | '<img src="/static/media/tree.svg" width="40">',
```
Comment (OleGiebel) [inline src/components/ConsumptionGraph.js]: This is the reset icon of the consumption graph, right? Why the change to the tree icon of the consumption calculation section?

Comment (VirginiaBalseiro-Zuehlke) [reply src/components/ConsumptionGraph.js]: Oh sorry! I was testing if the custom icons worked (they don't) and totally forgot to remove. I#ll remove and push again.

Comment (VirginiaBalseiro-Zuehlke) [reply src/components/ConsumptionGraph.js]: Done.

File: src/scss/css.scss [hunk 2]
```diff
@@ -143,4 +144,12 @@
         cursor: pointer;
     }
 
+}
+
+@media screen and (min-width: $breakpoint-small) {
+    .apexcharts-reset-zoom-icon,
+    .apexcharts-zoom-icon {
+        width: 55px
```
Comment (OleGiebel) [inline src/scss/css.scss]: These two classes are used on the div containers around the svg images. They do not change the size of the image itself.

Comment (VirginiaBalseiro-Zuehlke) [reply src/scss/css.scss]: The icons look bigger though.

Comment (OleGiebel) [reply src/scss/css.scss]: Just rechecked it and you are right. It does look bigger.
